packages/nimble-blazor/build/generate-hybrid/source/SprightBlazor.HybridWorkaround.js

```diff
@@ -0,0 +1,3 @@
+import { afterStarted } from '@ni/spright-blazor/wwwroot/SprightBlazor.lib.module';
```

Comment:
Can you undo this change. It creates an unnecessary coupling to the blazor build running first.